docs/development.md

```diff
@@ -150,7 +150,7 @@ make vendor-update
 4. Commit updated glide files and vendor
 
 ```bash
-git add glide.yaml glide.lock vendor
+git add glide.yaml glide.lock scripts/vendor-openshift.sh vendor
```

Comment:
Like I mentioned before, wouldn't this all be easier just to do a `git add .` and add a note here not not include and `.go` changes?

Reply:
I think that listing files explicitly like this is much safer.